### ISSUE TITLE
[HELP] How do I run TransRec Model?

### ISSUE THREAD

**Issue by karthikraja95** [shuyangli94/RecSysModels#2]
I am following the sample_pipeline notebook, I think till Train/Test/Validation split the procedure is the same.  Added prior to the validation and test set (u-p-i-j) format. I think I am doing something wrong in the sampling part I am not sure how to sample it properly? Any help is appreciated.

Here's my code:

'''
Run TransRec
'''
import tensorflow as tf
from recsys_models.models.transrec import TransRec
from recsys_models.pipeline import train_model

#### Set training parameters
max_epochs = 200
n_iterations = 1000
batch_size = 512
stopping_threshold = 1e-5

#### Get the validation and testing matrices
start = datetime.now()
validation_data = validation_df[['u', 'prior', 'i', 'j']].values
test_data = test_df[['u', 'prior', 'i', 'j']].values
print('{} - Generated u-p-i-j matrices for validation and testing'.format(
    datetime.now() - start
))

#### Initialize the graph
tf.reset_default_graph()
model = TransRec(n_users, n_items, k=5, lambda_emb=1e-4, lambda_bias=1e-4,
               opt_type=tf.contrib.opt.LazyAdamOptimizer, opt_args={'learning_rate': 0.007})
print('\n=== BEGIN Optimization for {} ==='.format(model.model_id))
print('    {} Max epochs, with early stoppage at {} Validation AUC change'.format(max_epochs, stopping_threshold))
print('    {} Iterations per epoch with {}-sized batches'.format(n_iterations, batch_size))

#### Open session and initialize graph weights
session = tf.Session()
session.run(tf.global_variables_initializer())

#### Train the model!
model, train_auc, validation_auc, test_auc = train_model(
    session, model, train_df, validation_data, test_data,
    n_iterations=n_iterations, batch_size=batch_size,
    min_epochs=10, max_epochs=max_epochs,
    stopping_threshold=stopping_threshold,
    sample_columns=['u', 'i'], column_order=['u', 'i', 'j'],
    n_items=n_items, items_by_user=train_items_by_user
)

#### Save model
suffix = '_transrec'
full_model_id = '{}{}'.format(model.model_id, suffix)
model_folder = os.path.join('tf_models', full_model_id)
if not os.path.exists(model_folder):
    os.makedirs(model_folder)
model.save(session, 'tf_models', suffix=suffix)
print('{} - Saved model to {}'.format(
    datetime.now() - start, model_folder
))

#### Cleanup
session.close()
gc.collect()
print()

Attached is the Full Traceback:

![Screen Shot 2019-10-21 at 9 35 49 AM](https://user-images.githubusercontent.com/40802179/67209946-3cd30200-f3e6-11e9-89b3-ac1968de003c.png)
